app/build.gradle

```diff
@@ -5,70 +5,88 @@
-		}
-		prod {
-			buildConfigField 'boolean', 'IS_DEV', 'false'
-		}
-	}
-
-	signingConfigs {
-		release {
-			storeFile file('testKeystore')
-			storePassword '123456'
-			keyAlias 'keyAlias'
-			keyPassword '123456'
-		}
-	}
-	buildTypes {
-		release {
-			minifyEnabled false
-			proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-			signingConfig signingConfigs.release
-		}
-	}
-
-	compileOptions {
-		sourceCompatibility = 1.8
-		targetCompatibility = 1.8
-	}
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        applicationId "org.dpppt.android.app"
+        minSdkVersion 23
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+    }
+
+    flavorDimensions "version"
+    productFlavors {
+        dev {
+            buildConfigField 'boolean', 'IS_DEV', 'true'
+        }
+        prod {
+            buildConfigField 'boolean', 'IS_DEV', 'false'
+        }
+    }
+
+    signingConfigs {
+        release {
+            storeFile file('testKeystore')
+            storePassword '123456'
+            keyAlias 'keyAlias'
+            keyPassword '123456'
+        }
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.release
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = 1.8
+        targetCompatibility = 1.8
+    }
 }
 
 dependencies {
-	implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    implementation 'org.dpppt:dp3t-sdk-android:0.1.2'
 
-	implementation 'org.dpppt:dp3t-sdk-android:0.1.2'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'androidx.fragment:fragment:1.2.4'
+    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0'
+    implementation 'androidx.lifecycle:lifecycle-livedata:2.2.0'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0'
+    implementation 'androidx.viewpager2:viewpager2:1.0.0'
 
-	implementation 'androidx.appcompat:appcompat:1.1.0'
-	implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-	implementation 'androidx.fragment:fragment:1.2.4'
-	implementation 'androidx.lifecycle:lifecycle-viewmodel:2.2.0'
-	implementation 'androidx.lifecycle:lifecycle-livedata:2.2.0'
-	implementation 'androidx.viewpager2:viewpager2:1.0.0'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.5'
 
-	implementation 'io.reactivex.rxjava3:rxandroid:3.0.0'
-	implementation 'io.reactivex.rxjava3:rxjava:3.0.0'
+    implementation "com.google.dagger:dagger:2.27"
+    kapt "com.google.dagger:dagger-compiler:2.27"
```

Comment:
You can define a version for groups of dependencies like in this case we can create a `daggerVersion` var